pkg/runner/banners.go

```diff
@@ -32,6 +32,20 @@ func (options *Options) normalRunTasks() {
 	if err != nil {
 		gologger.Fatalf("Could not read configuration file %s: %s\n", options.ConfigFile, err)
 	}
+
+	// If we have a different version of subfinder installed
+	// previously, use the new iteration of config file.
+	if configFile.Version != Version {
```

Comment:
This is a breaking change, I've lost my source keys from the config file after running subfinder from source on the master branch. Is this intentional?

Reply:
@vzamanillo i could not reproduce the issue. Could you please post steps you took? Since this works on my side.

Reply:
All it does is changes the configuration file's source, allsources and recursive-resources files without changing any other keys so i don't think there should be an issue.

Reply:
You are completely right, sorry for this, it was due to some changes in a local testing branch.

Reply:
@vzamanillo No problem, it happens all the time :P.